CONTRIBUTING.md

```diff
@@ -29,10 +29,16 @@ The following text equips you with knowledge that makes contributing to ilastik
    gh repo clone ilastik
    ```
 
+1. Configure conda to use [strict channel priority](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html#strict-channel-priority):
+
+   ```
+   conda config --set channel_priority strict
+   ```
+
 1. Install [conda-build][conda-build] in order to access the [conda develop][conda-develop] command:
```

Comment:
What do you think about moving to [`miniforge`](https://github.com/conda-forge/miniforge/#miniforge3)? I have no idea  whether it is stable enough though...

Reply:
Looks interesting, are you using it? I'm not. I'd wait until most of us are using it before recommending it.

Reply:
Me too, but use the default installed just out of the habit, and I'd rather use only conda-forge channel if possible.

Reply:
okay, I'll try to switch before Friday :_). Then let's maybe give it a go a bit by ourselves before recommending it. I definitely see the appeal.

Reply:
I tried to install a fresh ilastik env with miniforge, and it seems to work.